repository/src/main/java/org/alfresco/repo/event2/NodeResourceHelper.java

```diff
@@ -158,6 +160,15 @@ private boolean isSubClass(QName className, QName ofClassQName)
         return dictionaryService.isSubClass(className, ofClassQName);
     }
 
+    private String getAssocQName(NodeRef nodeRef) {
+        String result = null;
+        ChildAssociationRef primaryParent = nodeService.getPrimaryParent(nodeRef);
```

Comment:
This call never returns null, see org.alfresco.repo.node.db.DbNodeServiceImpl#getPrimaryParent

Reply:
done